ch1.md

```diff
@@ -19,26 +19,33 @@ I won't belabor you by listing each and every guideline I've heard throughout th
 Let's start with a touch of insanity. Here is a seagull application. When flocks conjoin they become a larger flock, and when they breed, they increase by the number of seagulls with whom they're breeding. Now, this is not intended to be good Object-Oriented code, mind you, it is here to highlight the perils of our modern, assignment based approach. Behold:
 
 ```js
-var Flock = function(n) {
-  this.seagulls = n;
-};
-
-Flock.prototype.conjoin = function(other) {
-  this.seagulls += other.seagulls;
-  return this;
-};
-
-Flock.prototype.breed = function(other) {
-  this.seagulls = this.seagulls * other.seagulls;
-  return this;
-};
-
-var flock_a = new Flock(4);
-var flock_b = new Flock(2);
-var flock_c = new Flock(0);
-
-var result = flock_a.conjoin(flock_c)
-    .breed(flock_b).conjoin(flock_a.breed(flock_b)).seagulls;
+// A Running Example: https://jsfiddle.net/1zhy31av/
+class Flock {
+  constructor(n) {
+    this.seagulls = n;
+  }
+
+  conjoin(other) {
+    this.seagulls += other.seagulls;
+    return this;
+  }
+
+  breed(other) {
+    this.seagulls = this.seagulls * other.seagulls;
+    return this;
+  }
+}
+
+const
+  flock_a = new Flock(4),
+  flock_b = new Flock(2),
+  flock_c = new Flock(0),
+  result =
+    flock_a
+      .conjoin(flock_c)
+      .breed(flock_b)
+      .conjoin(flock_a.breed(flock_b))
+      .seagulls;
```

Comment:
What about using `eslint` with the [airbnb-base](https://www.npmjs.com/package/eslint-config-airbnb-base) configuration to format & 'standardize' the examples and pieces of codes across the book ?

There's no real 'standard' in JavaScript (Golang I miss you <3), but I still believe that airbnb has one of the most commonly adopted. Using this as a baseline to resolve any formatting issue or questions about the way the code should be structured. For instance here:
```
  18:1  error  Split 'const' declarations into multiple statements   one-var
  19:3  error  Identifier 'flock_a' is not in camel case             camelcase
  20:3  error  Identifier 'flock_b' is not in camel case             camelcase
  21:3  error  Identifier 'flock_c' is not in camel case             camelcase
```